packages/kmod-5.10-nvidia/nvidia-tesla-ld.so.conf.in

```diff
@@ -0,0 +1 @@
+/usr/lib/nvidia/tesla/NVIDIA_VERSION/
```

Comment:
nit: it'd be somewhat better to use a token like `__LIBDIR__` here that we replace with the sed, to avoid hardcoding paths.

`packages/libbzip2/bzip2.pc.in` is an example of this.

I also don't love the trailing slash, since it seems obvious from context that it's a directory.